app/components/works/agreement_component.html.erb

```diff
@@ -1,6 +1,9 @@
 <div class="row justify-content-end agreement">
   <div class="form-check col-auto">
-    <%= form.check_box :agree_to_terms, required: true, class: 'form-check-input' %>
+    <%= form.check_box :agree_to_terms, required: true, class: 'form-check-input',
+        data: { action: "change->auto-citation#updateDisplay change->edit-deposit#check",
```

Comment:
I don't think `change->auto-citation#updateDisplay` is necessary, is it?  This doesn't contribute to the auto citation at all.

Reply:
oops, that was a mistake - i removed it, good catch